docs/guides/index.rst

```diff
@@ -46,3 +46,14 @@ These guides will help you customize or tune aspects of the Read the Docs build
     sitemaps
     specifying-dependencies
     wipe-environment
+
+
+Read the Docs for Business how-to guides
+----------------------------------------
+
+These guides are specific to :doc:`/commercial/index`.
+
+.. toctree::
+   :maxdepth: 1
+
+   private-submodules
```

Comment:
After this guide I'm planning to write a guide in how to install private pip packages, so this section isn't lonely. Or wherever we want to group guides that are only for .com